retrieval/target.go

```diff
@@ -359,9 +364,15 @@ func (t *Target) scrape(sampleAppender storage.SampleAppender) (err error) {
 	}()
 
 	for samples := range t.ingestedSamples {
-		for _, s := range samples {
-			s.Metric.MergeFromLabelSet(baseLabels, clientmodel.ExporterLabelPrefix)
-			sampleAppender.Append(s)
+		if t.proxy {
+			for _, s := range samples {
+				sampleAppender.Append(s)
```

Comment:
Never attaching an `instance` label so we can forget about the whole thing after ingesting a metric from PGW.
It makes inherently more sense to me to expose different modes to users even if they are doing almost exactly the same. Less edge cases for us, clearer configurations for the user, possibility for future adjustments without breaking or complicating configs.